doc/changes/changes_0.4.5.md

```diff
@@ -0,0 +1,12 @@
+# Extension Manager Interface 0.4.5, released 2025-02-12
+
+Code name: Fix boolean parameter serialization
+
+## Summary
+
+This release fixes and issue with the serialization of boolean parameters, that were serialized as in JSON as a 
+string value `"true"`/`"false"` instead of using the literal values `true`/`false`
```

Comment:
```suggestion
This release fixes an issue with the serialization of boolean parameters, that were serialized to JSON as a 
string value `"true"` / `"false"` instead of using the literal values `true` / `false`
```